cfme/tests/cloud/test_providers.py

```diff
@@ -727,42 +871,74 @@ def test_domain_id_validation(request, provider):
     """
-    Upload image copied from RHCF3-11271 to azure using powershell
-    similiar the script in the setup section.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1520196
 
     Polarion:
         assignee: anikifor
         casecomponent: Cloud
         caseimportance: medium
-        initialEstimate: 1/2h
-        setup: # Video Demo Script - Upload VHD to Azure
-               # You have to log in via Login-AzureRmAccount
-               Get-AzureRmSubscription -SubscriptionId "9ee63d8e-aee7-4121-861c-
-               d67a5b8d231e" -TenantId "2e64678d-2fa8-40ed-8922-c9b8e2c3f100" |
-               Select-AzureRmSubscription
-               $ResourceGroupName = "CFMEQE-Main"
-               $BlobLocation = "https://cfmeqestore.blob.core.windows.net/"
-               $BlobContainer = "templates"
-               $BlobName = "tmpl-osDisk.vhd"
-               $BlobDestination = $BlobLocation + $BlobContainer + "/" + $BlobName
-               $LocalFilePath = "C:\tmp\"
-               $LocalFileName = "cfme-azure-5.6.0.13-1.x86_64.vhd"
-               $LocalFile = $LocalFilePath + $LocalFileName
-               Add-AzureRmVhd -ResourceGroupName $ResourceGroupName -Destination
-               $BlobDestination -LocalFilePath $LocalFile -NumberOfUploaderThreads 8
+        initialEstimate: 1/6h
+        testSteps:
+            1. Add Network Security group on Azure with coma separated port ranges
+            `1023,1025` rule inbound/outbound ( ATM this feature is not allowed in
+            East US region of Azure - try West/Central)
+            2. Add such Azure Region into CFME
+            3. Refresh provider
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_provider_flavors_azure():
+    """
+    Verify that the vm flavors in Azure are of the correct sizes and that
+    the size display in CFME is accurate.  This test checks a regression
+    of https://bugzilla.redhat.com/show_bug.cgi?id=1357086
+    Low priority as it is unlikely to change once set.  Will want to check
+    when azure adds new sizes.  Only need to spot check a few values.
+    For current size values, you can check here:
+    https://azure.microsoft.com/en-us/documentation/articles/virtual-
+    machines-windows-sizes/
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Cloud
+        caseimportance: low
+        initialEstimate: 1/8h
         startsin: 5.6
-        title: Upload Azure image to Azure
     """
     pass
 
 
 @pytest.mark.manual
 @pytest.mark.tier(1)
-def test_create_azure_vm_from_azure_image():
+def test_market_place_images_azure():
     """
-    CFME image must be uploaded to Azure
+    https://bugzilla.redhat.com/show_bug.cgi?id=1491330
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/6h
+        testSteps:
+            1.Enable market place images
+            2.Verify the list of images
```

Comment:
Please split this to an expectedResults block, with a bit more detail on what is expected to be visible/not visible. Should enabling market place de-list the private images? Not sure from this test case.